hrms/hr/doctype/expense_claim/bulk_download.py

```diff
@@ -0,0 +1,172 @@
+import frappe
+from frappe import _
+from frappe.utils.csvutils import build_csv_response
+import pandas as pd
+from suite42_erp_app.suite42_erp_app.common.decorators import (
+    handle_exceptions_with_readable_message,
+)
+import json
+from datetime import datetime
+
+
+@frappe.whitelist()
+@handle_exceptions_with_readable_message
+def bulk_download():
```

Comment:
pass company and bank account as parameters

check company and bank accounts as One suite and kotak respectively